ts/Stock/Indicators/VBP/VBPOptions.d.ts

```diff
@@ -42,7 +42,7 @@ export interface VBPOptions extends SMAOptions {
 
 export interface VBPParamsOptions extends SMAParamsOptions {
     ranges?: number;
-    volumeSeriesID?: string;
+    volumeSeriesID: string;
```

Comment:
I'm curious about this one. This is true: to render the VbP indicator, `volumeSeriesID` must be set. The same for `linkedTo` option.

However, will that affect also `series.update({ ... });`? `volumeSeriesID` there is optional

Reply:
What do you think @TorsteinHonsi ?

Reply:
My though process with this one was: 
Since we set [the default value](https://github.com/highcharts/highcharts/blob/d1a37ffe3a085206c6d7b42d13381e3d6d0339db/ts/Stock/Indicators/VBP/VBPIndicator.ts#L154) for `volumeSeriesID`, does it make sense to leave it as an optional property in this interface?

Reply:
If TS is happy, I'm happy. The idea with `series.update()` and other update functions is that they tak `DeepPartial<SeriesOptions>`, which allows us to set non-optional properties on the default options. It is not consistently implemented yet, though.